README.md

```diff
@@ -155,7 +176,7 @@ POST /generate
 
 All configurations are saved in the [config.js](/config.json) file.  
 
-NOTE: In this document all `config.<obj>` variables refer to the data in the *config.json* file.
+> NOTE: In this document all `config.<obj>` variables refer to the data in the *config.json* file.
```

Comment:
Lets keep this simple, and remove `> `

Reply:
Generally `NOTE` is quoted. But, Okay! Done :relaxed: